Extension/src/Debugger/configurationProvider.ts

```diff
@@ -563,7 +563,7 @@ class WindowsConfigurationProvider extends DefaultConfigurationProvider {
         "ignoreFailures": true
     },
     {
-        "description":  "${localize("enable.intel.disassembly.flavor", "Set Disassembly Flavor to {0}", "Intel").replace(/\"/g, "\\\"")},
+        "description":  "${localize("enable.intel.disassembly.flavor", "Set Disassembly Flavor to {0}", "Intel").replace(/\"/g, "\\\"")}",
```

Comment:
There's a lot of double quotes in there.  Can we change the strings to use single quotes in this case?

Reply:
And, I'm just referring to the outermost quotes to change to single quotes

Reply:
Why? I don't recall seeing JSON with single quotes....that might only work with TypeScript/JavaScript strings?

Reply:
The whole string is part of a `` string literal.

Reply:
Its similar to the `description` field above. 

One way to fix this string mess is to move it to be a javascript object and ToString it to the caller.

Reply:
My bad. I thought this was a JavaScript object.  No need to make changes then. I already approved.